execution_engine/src/shared/host_function_costs.rs

```diff
@@ -63,7 +63,7 @@ const DEFAULT_RET_VALUE_SIZE_WEIGHT: u32 = 420;
 const DEFAULT_REVERT_COST: u32 = 500;
 const DEFAULT_SET_ACTION_THRESHOLD_COST: u32 = 74_000;
 const DEFAULT_TRANSFER_FROM_PURSE_TO_ACCOUNT_COST: u32 = 160_000;
-const DEFAULT_TRANSFER_FROM_PURSE_TO_PURSE_COST: u32 = 82_000;
+pub const DEFAULT_TRANSFER_FROM_PURSE_TO_PURSE_COST: u32 = 82_000;
```

Comment:
Seems pretty odd to have a single pub in a sea of non-pubs just for the sake of a test.  Why would we not have similar tests for the others?

Reply:
Removed pub here and just used a custom value that doesnt rely on defaults.